bundle.yaml

```diff
@@ -14,10 +15,44 @@ applications:
     charm: grafana-k8s
     channel: edge
     scale: 1
+  loki:
+    charm: loki-k8s
+    channel: edge
+    scale: 1
 
 relations:
-- - prometheus
-  - alertmanager
-- - grafana
-  - prometheus
+- - prometheus:alertmanager
+  - alertmanager:alerting
+- - grafana:grafana-source
+  - prometheus:grafana-source
+#- - grafana:grafana-source
+#  - loki:grafana-source
+
+--- # overlay.yaml
+# Ref: https://discourse.charmhub.io/t/bundle-reference/1158
+applications:
+  alertmanager:
+    # This (offers or acl?) is a deployment-specific field and can only be specified as part of an overlay.
+    offers:
+      alertmanager:
+        endpoints:
+          - alerting
```

Comment:
I do not see a reason why we should expose the alerting endpoint. Alerts are generated exclusively by Prometheus and Loki at this stage.